web/src/components/InformationCards.tsx

```diff
@@ -0,0 +1,81 @@
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { MediaQueries } from "@/lib/PageSizes";
+import { useMediaQuery } from "@mui/material";
+import Link from "next/link";
+import { ReactElement } from "react";
+
+const renderCard = (
+  color: string,
+  headerLine1: string,
+  headerLine2: string,
+  supportingText: string,
+  buttonLink: string,
+  buttonText: string,
+  buttonStyleProp: string,
+  intercom?: boolean
+) => (
+  <div className={`landing-card border-${color} border-top-105`}>
+    <div className="flex flex-column fac space-between padding-x-2 padding-y-6 height-full">
+      <div className="text-center">
+        <h3 className="margin-0">{headerLine1}</h3>
+        <h3 className="margin-bottom-3">{headerLine2}</h3>
+        <div>{supportingText}</div>
+      </div>
+      <div>
+        <Link href={buttonLink} passHref>
+          <button
+            className={`usa-button usa-button margin-right-0 btn-${buttonStyleProp} ${
+              intercom ? "intercom-button" : ""
+            }`}
+          >
+            {buttonText}
+          </button>
+        </Link>
+      </div>
+    </div>
+  </div>
+);
+
+const InformationCards = (): ReactElement => {
+  const { Config } = useConfig();
+  const isDesktopAndUp = useMediaQuery(MediaQueries.desktopAndUp);
+  const cardConfig = Config.landingPage;
+  return (
+    <div className={`${isDesktopAndUp ? "flex flex-row fja" : ""}`}>
+      {renderCard(
+        "primary",
+        cardConfig.card1HeaderLine1,
+        cardConfig.card1HeaderLine2,
+        cardConfig.card1SupportingText,
+        cardConfig.card1ButtonLink,
+        cardConfig.card1Button,
+        "primary"
+      )}
+      <div className={`${isDesktopAndUp ? "desktop:margin-x-3" : "margin-y-2"}`}>
+        {renderCard(
+          "accent-cooler",
+
+          cardConfig.card2HeaderLine1,
+          cardConfig.card2HeaderLine2,
+          cardConfig.card2SupportingText,
+          cardConfig.card2ButtonLink,
+          cardConfig.card2Button,
+          "accent-cooler"
+        )}
+      </div>
+      {renderCard(
+        "info-dark",
+
+        cardConfig.card3HeaderLine1,
+        cardConfig.card3HeaderLine2,
+        cardConfig.card3SupportingText,
+        "",
+        cardConfig.card3Button,
+        "info",
+        true
+      )}
+    </div>
+  );
+};
+
+export default InformationCards;
```

Comment:
As a pattern, we don't do `export default` on any components except for Next page components (and there, we only do it because Next requires it)

For everything else, we do `export const ComponentName ...` inline when we declare it